### PR TITLE
[VAN-2020] Add one organizer and add link for event twitter

### PR DESCRIPTION
**Pull request by Velutas**
One organizer is added and the link for twitter is no longer commented out.

### CODE REVIEW

**Comment by adrianmoisey** [comment]
Can you please email info@devopsdays.org with Alicja's email address?
We prefer to add people to the website, slack and mail group all at the same time, just to keep things in sync.
Once you've done that I can merge this PR.

File: data/events/2020-vancouver.yml [hunk 1]
```diff
@@ -69,6 +69,7 @@ team_members: # Name is the only required field for team members.
   - name: "Samantha Toner"
   - name: "Kyle Young"
     twitter: "ksgyoung"
+  - name: "Alicja Raszkowska"
```
Comment (ggunson) [inline data/events/2020-vancouver.yml]: Please add twitter `mamrotynka` (I forgot we had that). Thanks.

Comment (adrianmoisey) [reply data/events/2020-vancouver.yml]: Fixed this.

**Comment by ggunson** [comment]
Hi @adrianmoisey 

Just FYI I noted the twitter account as something for @Velutas to edit, not you. I can be more explicit next time, and save you from doing work. Thanks for everything!

**Pull request merged**